### ISSUE TITLE
Update sidebar icon?

### ISSUE THREAD

**Issue by JJJ** [EverythingSmartHome/everything-presence-addons#13]
The sidebar icon appears to be using the fallback puzzle piece.

Would you be open to picking something specific? Maybe `mdi:couch` or `mdi:bed`?

![jjj-2024-11-08-at-19-39-25-UTC@2x](https://github.com/user-attachments/assets/caa1fbf3-a332-41cb-abf3-fb941b6cbfdb)

**Comment by JJJ** [comment]
Related to #11.

**Comment by noxhirsch** [comment]
Yes, please - but I'd suggest `mdi:motion-sensor` or `mdi:floor-plan`

**Comment by jcrowegitHu8** [comment]
Yes, please.  I feel that ```mdi:view-grid-plus-outline``` better represents what the configurator does.  You are just drawing boxes ;)

![Image](https://github.com/user-attachments/assets/e9f9926b-84a8-412e-a100-4d5a9470cff5)
